packages/editor/templates/Crypto Pizzas/README.md

```diff
@@ -54,20 +54,20 @@ And that's all the code relevant to this dapp, there are other functions for glu
 > Find the CSS file in _app/app.css_
 > Find the JavaScript file in _app/app.js_
 
-### 1. Configure
+### Configure
```

Comment:
Let's keep the numbering here to stay consistent with the other template READMEs